web/docs/tql2/operators.md

```diff
@@ -59,8 +65,10 @@ Operator | Description | Example
 
 Operator | Description | Example
 :--------|:------------|:-------
-[`diagnostics`](./operators/diagnostics.md) | Returns diagnostic events of managed pipelines | `diagnostics`
+[`from`](./operators/from.md) | Read events from an URI <br/> Create events ad-hoc| `from "https://example.com/file.json.gz"` <br/> `from {key: "value"}…` <!--at the top because its important-->
```

Comment:
1. Should be "Reads" and "Creates". 
2. There must be a better alternative to "ad-hoc".
3. Can you use a shorter URL in the example so that it doesn't linebreak (on a MacBook)?
4. Do we really want to have a linebreak here? I see why you did it, but it looks weird.

Reply:
1-3: Done.
Regarding 4: I see only three options:
* Leave out the 2nd usage entirely
* Document them on the same line, with a semicolon (?) in between descriptions/codeblocks
* Document them on separate lines as is.